acceptable/_oas.py

```diff
@@ -0,0 +1,45 @@
+"""
+Helpers to translate acceptable metadata to OpenAPI specifications (OAS).
+"""
+
+import sys
+from typing import Any
+
+import yaml
+
+
+def to_dict(source: Any):
```

Comment:
This might be a good candidate for a leading underscore.

Reply:
Done.